<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 45 oblique 15px &quot;URW Gothic L&quot;;
</string>
    </property>
    <property name="text">
     <string>Debug Console</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>22</y>
      <width>541</width>
      <height>140</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="Card">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(176, 170, 255);
border: 2px solid #6799FF;
border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;
</string>
       </property>
       <property name="text">
        <string>Card</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="home">
       <property name="styleSheet">
        <string notr="true">
background-color:#B2CCFF; 

border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;

</string>
       </property>
       <property name="text">
        <string>home</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Moving">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(176, 170, 255);
border: 2px solid #6799FF;
border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;
</string>
       </property>
       <property name="text">
        <string>Moving</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="Position">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(176, 170, 255);
border: 2px solid #6799FF;
border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;
</string>
       </property>
       <property name="text">
        <string>Position State</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="failed">
       <property name="styleSheet">
        <string notr="true">
background-color:#B2CCFF; 

border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;


</string>
       </property>
       <property name="text">
        <string>failed</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="stopped">
       <property name="styleSheet">
        <string notr="true">
background-color:#B2CCFF; 

border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;

</string>
       </property>
       <property name="text">
        <string>stopped</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="succeeded">
       <property name="styleSheet">
        <string notr="true">
background-color:#B2CCFF; 

border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;

</string>
       </property>
       <property name="text">
        <string>succeeded</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Work">
       <property name="styleSheet">
        <string notr="true">background-color:rgb(176, 170, 255);
border: 2px solid #6799FF;
border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;
</string>
       </property>
       <property name="text">
        <string>Work</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="moving">
       <property name="styleSheet">
        <string notr="true">
background-color:#B2CCFF; 

border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;

</string>
       </property>
       <property name="text">
        <string>moving</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="admin">
       <property name="styleSheet">
        <string notr="true">
background-color:#B2CCFF; 

border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;

</string>
       </property>
       <property name="text">
        <string>admin</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="table">
       <property name="styleSheet">
        <string notr="true">
background-color:#B2CCFF; 

border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;

horizontalAlignment: AlignHCenter;</string>
       </property>
       <property name="text">
        <string>table</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="serve">
       <property name="styleSheet">
        <string notr="true">
background-color:#B2CCFF; 

border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;

</string>
       </property>
       <property name="text">
        <string>serve</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="mmoving">
       <property name="styleSheet">
        <string notr="true">
background-color:#B2CCFF; 

border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;

</string>
       </property>
       <property name="text">
        <string>moving</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="rest">
       <property name="styleSheet">
        <string notr="true">
background-color:#B2CCFF; 

border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;

font:center;</string>
       </property>
       <property name="text">
        <string>rest</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="order">
       <property name="styleSheet">
        <string notr="true">
background-color:#B2CCFF; 

border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;

</string>
       </property>
       <property name="text">
        <string>order</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="interrupted">
       <property name="styleSheet">
        <string notr="true">
background-color:#B2CCFF; 

border-radius:20px;
color:#FFFFFF;
font-size:20px;
font: 45 oblique 20px &quot;URW Gothic L&quot;;

</string>
       </property>
       <property name="text">
        <string>interrupted</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>71</width>
      <height>122</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="bettery4">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(199, 199, 199); 
border: 4px solid #FFFFFF;
}

QPushButton:pressed{
background-color:rgb(99, 255, 169); 
border: 4px solid #FFFFFF;

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bettery3">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(199, 199, 199); 
border: 4px solid #FFFFFF;
}

QPushButton:pressed{
background-color:rgb(99, 255, 169); 
border: 4px solid #FFFFFF;

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bettery2">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(199, 199, 199); 
border: 4px solid #FFFFFF;
}

QPushButton:pressed{
background-color:rgb(99, 255, 169); 
border: 4px solid #FFFFFF;

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bettery1">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(199, 199, 199); 
border: 4px solid #FFFFFF;
}

QPushButton:pressed{
background-color:rgb(99, 255, 169); 
border: 4px solid #FFFFFF;

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbBettery">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 45 oblique 15px &quot;URW Gothic L&quot;;
</string>
    </property>
    <property name="text">
     <string>Battery</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>71</width>
      <height>122</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="v4">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(199, 199, 199); 
border: 4px solid #FFFFFF;
}

QPushButton:pressed{
background-color:rgb(252, 147, 255); 
border: 4px solid #FFFFFF;

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="v3">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(199, 199, 199); 
border: 4px solid #FFFFFF;
}

QPushButton:pressed{
background-color:rgb(252, 147, 255); 
border: 4px solid #FFFFFF;

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="v2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(199, 199, 199); 
border: 4px solid #FFFFFF;
}

QPushButton:pressed{
background-color:rgb(252, 147, 255); 
border: 4px solid #FFFFFF;

}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="v1">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color:rgb(199, 199, 199); 
border: 4px solid #FFFFFF;
}

QPushButton:pressed{
background-color:rgb(252, 147, 255); 
border: 4px solid #FFFFFF;

}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbVelocity">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 45 oblique 15px &quot;URW Gothic L&quot;;
</string>
    </property>
    <property name="text">
     <string>Velocity</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbBetteryC">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 45 oblique 15px &quot;URW Gothic L&quot;;
</string>
    </property>
    <property name="text">
     <string>Battery</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbVelocityC">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>230</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 45 oblique 15px &quot;URW Gothic L&quot;;
</string>
    </property>
    <property name="text">
     <string>Velocity</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(217,229,255);
border-image: url(:/transparent.png);
image: url(:/qt-logo.ico);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
